resources/qml/pages/MessagingPage.qml

```diff
@@ -324,8 +436,7 @@ Page {
                 if (editbox.text.length === 0 && sendmsgview.attachmentPath.length === 0 && shmoose.canSendFile()) {
                     sendmsgview.attachmentPath = ""
                     fileModel.searchPath = shmoose.settings.ImagePaths
-                    pageStack.push(pageImagePicker)
-                    pageImagePicker.selected.connect(processAttachment)
+                    pageStack.push(shmoose.settings.SendOnlyImages ? imagePickerPage: filePickerPage)
```

Comment:
This are Sailfish deliverd pickers - rigtht? So ImagePickerPage.qml can be removed?

Reply:
Yes. Do you want me to remove it?

Reply:
Yes, please. There should not be any file which is not used any more.

Reply:
Picker removed